src/go/rpk/build.sh

```diff
@@ -9,12 +9,23 @@
 # by the Apache License, Version 2.0
 
 version=$1
+root=$(git rev-parse --show-toplevel)
 rev=${2:-$(git rev-parse --short HEAD)}
 
-out_dir="$(go env GOOS)-$(go env GOARCH)"
+go_bin="${root}/vbuild/go/bin/go"
+if [ ! -f "${go_bin}" ]; then
+    # try using the system go
+    if ! command -v go &> /dev/null
+    then
+       echo "Please either install go systemwide or run vtools install go-compiler"
+       exit
+    fi
+    go_bin=$(which go)
+fi
+out_dir="$($go_bin env GOOS)-$($go_bin env GOARCH)"
 
 mkdir -p ${out_dir}
 
 pkg='vectorized/pkg/cli/cmd/version'
 
-go build -ldflags "-X ${pkg}.version=${version} -X ${pkg}.rev=${rev}" -o ${out_dir} ./...
+$go_bin build -ldflags "-X ${pkg}.version=${version} -X ${pkg}.rev=${rev}" -o ${out_dir} ./...
```

Comment:
i wonder if this is for both, local and for CI. if it's for CI, it might not work. if it's for local hacking, then yeah this is good.

Reply:
good point, it passes CI, @0xdiba opinions?

Reply:
adding @0x5d  for his thoughts

Reply:
is this necessary for this pr?

Reply:
not really lets drop it